cmds/wifi-ap/cmd_wifi-ap.go

```diff
@@ -68,6 +70,17 @@ func main() {
 		err = wifiAPClient.Enable()
 	case opts.disable:
 		err = wifiAPClient.Disable()
+	case opts.enabled:
+		res, err := wifiAPClient.Enabled()
+		if err != nil {
+			fmt.Println("Error:", err)
```

Comment:
this error is actually done in the end of the method for any option. Maybe this could be like
var res bool
res, err = wifiAPClient.Enabled()
if res {
	fmt.Println("Wifi-ap is UP")
} else {
	fmt.Println("Wifi-ap is Down")
}